minecraft.js

```diff
@@ -96,12 +115,33 @@ function shutdownMinecraft() {
 function restartMinecraft() {
   if (serverStatus == MINECRAFT_STATUS.RUNNING) {
     shutdownMinecraft();
-    startMinecraft();
-  } else {
-    startMinecraft();
   }
+  startMinecraft();
 }
 
 function getMinecraftStatus() {
   return serverStatus;
 }
+
+function fileToJSON(input) {
+  var output = {};
+
+  input.split('\n').forEach(function (line) {
+    if (line[0] === '#') {
```

Comment:
You should probably also handle empty lines. So if the line is an empty string, skip as well.

Reply:
That use case has already been added - it's the one you commented on below.